app/src/sharedTest/java/org/oppia/android/app/ongoingtopiclist/OngoingTopicListActivityTest.kt

```diff
@@ -111,6 +111,10 @@ class OngoingTopicListActivityTest {
       profileId = profileId,
       timestampOlderThanOneWeek = false
     )
+    storyProfileTestHelper.markCompletedRatiosStory1Exp0(
```

Comment:
Can you clarify why this is needed now?

Reply:
Yes, sure.
Actually, in `OngoingTopicListActivityTest` we want some topics to be present in the `OngoingTopicList` so that, we can perform tests on them. Earlier, before this logic update, a `Topic` (having two stories), and if only one story was completed then that was considered as `InProgress`. So completing the first story of `Ratios and Proportional Reasoning` was enough to include it in the `OngoingTopicList`. But now after the updated logic, we know that completing one story will mark the `Topic` as `Completed`, so here we started another story of this topic and completed only its first chapter so that the second story is Ongoing and the `Topic` gets included in the `OngoingTopicList`.

Reply:
Done